polaris-react/src/components/IndexFilters/tests/IndexFilters.test.tsx

```diff
@@ -394,5 +428,72 @@ describe('IndexFilters', () => {
         disabled: true,
       });
     });
+
+    it('renders EditColumnsButton with the disabled prop', () => {
+      const wrapper = mountWithApp(
+        <IndexFilters {...defaultProps} disabled showEditColumns />,
+      );
+
+      expect(wrapper).toContainReactComponent(EditColumnsButton, {
+        disabled: true,
+      });
+    });
+  });
+
+  describe('EditColumnsButton', () => {
+    it('renders when showEditColumns is true', () => {
```

Comment:
(update prop name in test file as well)

```suggestion
    it('renders when showEditColumnsButton is true', () => {
```